package di.unito.Exceptions

import io.sarl.core.Failure

event ParseException extends Failure {

	var e : Exception

}
